interfaces/src/db.ts

```diff
@@ -105,11 +105,10 @@ export interface Query {
 
 // tslint:disable-next-line:no-empty-interface
 export interface ClientContext {
+  appId: string;
   auth: {
-    v1: {
-      // Contents TBD
-      appId: string;
-      apiKey: string;
+    v1?: {
```

Comment:
If this is optional then you needn't include it everywhere.

Reply:
It's optional so you can use `v2` instead of `v1` once we specify it.